dist/fluxo_react_connect_stores.js

```diff
@@ -57,9 +83,11 @@
         var canceler =
           store.on(
             ["change", "stores:change"],
-            this.updateStateByStore.bind(null, storeName)
+            this.queueRenderRequest.bind(null, storeName)
           );
 
+        this.stores[storeName] = store;
+
```

Comment:
@samuelsimoes why keep a reference on `this.stores` since we already have `stores` given to the `FluxoReactConnectStores` function?